app/views/docs/oauth-providers/github.md

```diff
@@ -346,4 +346,7 @@ let account = Account(client)
 let session = try await account.updateSession(
     sessionId: "current"
 )
-```
\ No newline at end of file
+```
+## Accessing the Provider Access Token
+
+After the user has been authorized and redirected back to your app, you can find their GitHub provider access token in the [session object](https://appwrite.io/docs/models/session) by accessing the `providerAccessToken` property. This access token can be used to make requests to the GitHub API on their behalf.
```

Comment:
I think more context is needed here, why would someone need to use this access token? I would start by explaining what they are.